experiment_name: "square_path_scenario"
# if another world is used, add the world file to benchmarking_tool/world and provide the name of the new yaml file
world_name: 'scenario_world.world'
# if another map is used, add the map files to benchmarking_tool/map and provide the name of the new yaml file
map_name: 'map.yaml' 
# add gazebo model path, if multiple path, please spereate them by a :
models_path: ''

# Choose one of rhee following controllers, by adding its name to the controller type list

#DWB
#RPP
#RPP_RSC
#DWB_RSC

# if another controller is used, add the configuration file to turtlebot3_navigation2/param and provide the name of the new yaml file
controller_type: ['DWB', 'RPP']   #name of behaviour tree 
#Add the name of the robot navigation configuration, this file should be placed at turtlebot3/turtlebot3_navigation2
nav_config: 'burger'
#Enter the  required model of turtlebot3
#burger
#waffle
#waffle_pi
turtlebot3_model: 'burger'
#provide the pose in a decimal form
spawn_pose_x: 2.5
spawn_pose_y: -4.25
spawn_pose_yaw: 0.026  #radians 


# Specify either to send one goal, specific trajectory, or several waypoints 
# square, circle, several_waypoints, one_goal
trajectory_type: 'square'
#square parameter 
side_length: 2.5


